### ISSUE TITLE
Modify esgf.properties to use dots instead of underscores

### ISSUE THREAD

**Issue by William-Hill** [ESGF/esgf-installer#328]
I modified the keys in esgf.properties to use underscores instead of dots to be more Pythonic.  But during the installation of CoG, I discovered that the setup script is hardcoded to look for key names with dots.  I may have to change back to dots to avoid breaking the CoG setup.

**Comment by William-Hill** [comment]
Reference: _setupConfig() in at https://github.com/EarthSystemCoG/COG/blob/568f15de5ec865a84bc88bf18d9db2c22ae87317/cog/installation/config.py

**Comment by LucaCinquini** [comment]
Hi William,
not only - I think many applications refer to those keys by their traditional name, which includes the “.” - changing the key names is likely to break many things…
Thanks, L

On Jan 25, 2018, at 5:16 PM, William-Hill <notifications@github.com<mailto:notifications@github.com>> wrote:


I modified the keys in esgf.properties to use underscores instead of dots to be more Pythonic. But during the installation of CoG, I discovered that the setup script is hardcoded to look for key names with dots. I may have to change back to dots to avoid breaking the CoG setup.
